src/components/paypalMeDataPropTypes.js

```diff
@@ -1,15 +0,0 @@
-import PropTypes from 'prop-types';
-
-export default PropTypes.shape({
-    /** This is always 'PayPal.me' */
-    title: PropTypes.string,
-
-    /** The paypalMe address */
-    description: PropTypes.string,
-
-    /** This is always 'payPalMe' */
-    methodID: PropTypes.string,
-
-    /** This is always 'payPalMe' */
-    accountType: PropTypes.string,
-});
```

Comment:
@marcochavezf Do you have context on why this was added? I think it was not added that long time ago and curious if there might be some side-effects of removing it.

Reply:
Ah yeah, it was introduced in this [PR](https://github.com/Expensify/App/pull/10447/files#diff-eec5ba837852ed6fbe3e2ee4fe371c45b643c59932f6abe38aaf665329a74219). The problem is that we were only storing only the Paypal address in a different Onyx key [here](https://github.com/Expensify/App/pull/10447/files#diff-93c1bb3a403f942b2809d43de01f06366a9fed3060e3894959c268b53eb97be8R275-R289). So the idea is to use only one key (`NVP_PAYPAL_ME_ADDRESS`) as the source of truth.

Reply:
I see so if there is not a good reason to keep the payPal nvp shape which was introduced in that PR we should ditch it and use the payPalMeAddress one

That seems cleaner and that is how it was before